src/functions/findFirst.js

```diff
@@ -78,6 +93,32 @@ export const findFirst = wrap({
         }
         return undefined;
     },
+    tests: process.env.NODE_ENV !== "development" ? undefined : {
+        "basicUsage": (hi) => {
+            const seq = hi("a string to be searched").findFirst("to be");
+            hi.assert(seq, (r) => r.index === 9);
```

Comment:
Great ideas here. I'm awful at the basic usage tests lol. Tests as documentation is tricky for me, which is probably a good thing.